Documentation Standard Updates (#8)
fivetran/dbt_twilio: README.md

# Twilio Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_twilio/))
## What does this dbt package do?

- Produces modeled tables that leverage Twilio data from [Fivetran's connector](https://fivetran.com/docs/applications/twilio) in the format described by [this ERD](https://fivetran.com/docs/applications/twilio#schemainformation) and builds off the output of our [Twilio source package](https://github.com/fivetran/dbt_twilio_source).

> The following table provides a detailed list of all models materialized within this package by default.
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_twilio/#!/overview?g_v=1).

Author: ```suggestion
<!--section=“twilio_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_twilio/#!/overview?g_v=1).
```

> | **Model**                          | **Description**                                                                                                                                                                                                                              |

Author: ```suggestion
| **Table**                          | **Description**                                                                                                                                                                                                                              |
```

- Produces modeled tables that leverage Twilio data from [Fivetran's connector](https://fivetran.com/docs/applications/twilio) in the format described by [this ERD](https://fivetran.com/docs/applications/twilio#schemainformation) and builds off the output of our [Twilio source package](https://github.com/fivetran/dbt_twilio_source).

<!--section=“twilio_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_twilio/#!/overview?g_v=1).

| **Table**                          | **Description**                                                                                                                                                                                                                              |
|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | [twilio__message_enhanced](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__message_enhanced)    | This model provides additional information of every message sent or received.                                                                                                         |

Reviewer: ```suggestion
| [twilio__message_enhanced](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__message_enhanced)    | This table provides additional information of every message sent or received.                                                                                                         |
```

> | [twilio__number_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__number_overview)      | This model has aggregate messaging information for each phone number level, such as total messages, total inbound messages, total messages by status, and total spend.                                                                                                         |

Reviewer: ```suggestion
| [twilio__number_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__number_overview)      | This table has aggregate messaging information for each phone number level, such as total messages, total inbound messages, total messages by status, and total spend.                                                                                                         |
```

> | [twilio__account_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__account_overview)      | This model provides aggregate information per each account regarding the Twilio Messages resource. |                                                     |

Reviewer: ```suggestion
| [twilio__account_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__account_overview)      | This table provides aggregate information per each account regarding the Twilio Messages resource. |                                                     |
```

| [twilio__number_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__number_overview)      | This model has aggregate messaging information for each phone number level, such as total messages, total inbound messages, total messages by status, and total spend.                                                                                                         |
| [twilio__account_overview](https://fivetran.github.io/dbt_twilio/#!/model/model.twilio.twilio__account_overview)      | This model provides aggregate information per each account regarding the Twilio Messages resource. |                                                     |

> ## How do I use the dbt package?

Author: ```suggestion
<!--section-end-->

## How do I use the dbt package?
```
